starter-code/index.html

```diff
@@ -3,40 +3,121 @@
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+
+<header class="header-main-div">
+      <div class="header-logo-img">
+        <img src="images/spotify-logo.png" alt="spotify-logo">
+      </div>
+      <div class="header-ul">
+        <ul>
+          <li>Premium</li>
+          <li>Discover </li>
+          <li>Help</li>
+          <li>Download</li>
+        </ul>
+      </div>
+</header>
+
+
+<main>
+
+<section class="main-section-one">
+<div>
+  <h1>Music for everyone.</h1>
+</div>
+<div>
+  <h3>Spotify is now free on mobile, tablet and computer.
+      Listen to the right music, wherever you are.</h3>
+</div>
+
+</section>
+    
+
+
+<section class="main-section-two" >
+  <div class="section-two-title">
+    <h1>What’s on Spotify?</h1>
+  </div>
+  <div class="section-two-info">
+    <div class="section-two-inner-divs">
+      <img src="images/music-icon.png" alt="music-icon">
+      <h3>Millions of Songs</h3>
+      <p>There are millions of
+          songs on Spotify</p>
+    </div>
+    <div class="section-two-inner-divs" >
+      <img  src="images/high-quality-icon.png" alt="quality-icon">
+      <h3>HD Music</h3>
+      <p>Listen to music as if you
+          were listening live</p>
+    </div>
+    <div class="section-two-inner-divs">
+      <img src="images/devices-icon.png" alt="devices-icon">
+      <h3>Stream Everywhere</h3>
+      <p>Stream music on your
+        smartphone, tablet or
+        computer</p>
+    </div>
+
+  </div>
+</section>
+
+
+
+<section class="section-three-main">
+  <div class="section-three-background">
+    <div class="s3-div1">
+        <div class="s3-subdiv1">
+            <div id="kanye-div">
+              <h1>It’s as yeezy as Kanye West.</h1>
+            </div>
+            <div class="s3-inner-divs">
+              <h3>Search</h3>
+              <p>Know what you want to listen to?
+                Just search and hit play.</p>
+            </div>
+            <div class="s3-inner-divs">
+              <h3>Browse</h3>
+              <p>Check out the latest charts,
+                  brand new releases and great
+                  playlists for right now.</p>
+            </div>
+            <div class="s3-inner-divs">
+              <h3>Discover</h3>
+              <p>Enjoy new music every Monday
+                  with your own personal playlist.
+                  Or sit back and enjoy Radio.</p>
+            </div>
+        </div>
+        <div class="s3-subdiv2">
+            <div>
+              <img id="spotify-icon-final" src="images/spotify-icon-white.png" alt="spotify-icon">
+            </div>
+        </div>
+    </div>
+     <div class="s3-div2">
+        <div>
+          <img id="kanye-icon" src="images/spotify-app.jpg" alt="spotify-app">
+        </div>
+     </div>  
+  </div>
+
+</section>
+
```

Comment:
No tenéis que dejar espacios vacíos sin razón para que vuestro código se vea bien currado y ordenado y limpio.